Gemfile.lock

```diff
@@ -1,62 +1,60 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    better_errors (2.1.1)
+    better_errors (2.5.0)
       coderay (>= 1.0.0)
-      erubis (>= 2.6.6)
+      erubi (>= 1.0.0)
       rack (>= 0.9.0)
-    byebug (9.0.6)
-    coderay (1.1.1)
-    commander (4.4.3)
-      highline (~> 1.7.2)
-    database_cleaner (1.6.1)
+    byebug (10.0.2)
+    coderay (1.1.2)
+    commander (4.4.7)
+      highline (~> 2.0.0)
+    database_cleaner (1.7.0)
     diff-lcs (1.3)
-    erubis (2.7.0)
-    event_sourcery (0.13.0)
-    event_sourcery-postgres (0.3.0)
-      event_sourcery (>= 0.10.0)
+    erubi (1.7.1)
+    event_sourcery (0.22.0)
+    event_sourcery-postgres (0.8.0)
+      event_sourcery (>= 0.14.0)
       pg
-      sequel (~> 4.38)
-    highline (1.7.8)
-    method_source (0.8.2)
-    mustermann (1.0.0)
+      sequel (>= 4.38)
+    highline (2.0.0)
+    method_source (0.9.2)
+    mustermann (1.0.3)
     pg (0.20.0)
-    pry (0.10.4)
+    pry (0.12.2)
       coderay (~> 1.1.0)
-      method_source (~> 0.8.1)
-      slop (~> 3.4)
-    pry-byebug (3.4.2)
-      byebug (~> 9.0)
+      method_source (~> 0.9.0)
+    pry-byebug (3.6.0)
+      byebug (~> 10.0)
       pry (~> 0.10)
-    rack (2.0.3)
-    rack-protection (2.0.0)
+    rack (2.0.6)
+    rack-protection (2.0.4)
       rack
-    rack-test (0.6.3)
-      rack (>= 1.0)
-    rake (12.0.0)
-    rspec (3.6.0)
-      rspec-core (~> 3.6.0)
-      rspec-expectations (~> 3.6.0)
-      rspec-mocks (~> 3.6.0)
-    rspec-core (3.6.0)
-      rspec-support (~> 3.6.0)
-    rspec-expectations (3.6.0)
+    rack-test (1.1.0)
+      rack (>= 1.0, < 3)
+    rake (12.3.1)
+    rspec (3.8.0)
+      rspec-core (~> 3.8.0)
+      rspec-expectations (~> 3.8.0)
+      rspec-mocks (~> 3.8.0)
+    rspec-core (3.8.0)
+      rspec-support (~> 3.8.0)
+    rspec-expectations (3.8.2)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-mocks (3.6.0)
+      rspec-support (~> 3.8.0)
+    rspec-mocks (3.8.0)
       diff-lcs (>= 1.2.0, < 2.0)
-      rspec-support (~> 3.6.0)
-    rspec-support (3.6.0)
-    sequel (4.47.0)
+      rspec-support (~> 3.8.0)
+    rspec-support (3.8.0)
+    sequel (5.14.0)
     shotgun (0.9.2)
       rack (>= 1.0)
-    sinatra (2.0.0)
+    sinatra (2.0.4)
       mustermann (~> 1.0)
       rack (~> 2.0)
-      rack-protection (= 2.0.0)
+      rack-protection (= 2.0.4)
       tilt (~> 2.0)
-    slop (3.6.0)
-    tilt (2.0.7)
+    tilt (2.0.8)
```

Comment:
### [tilt](http://github.com/rtomayko/tilt/)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 2.0.7 → 2.0.8

[_~~change-log~~, [source-code](http://github.com/rtomayko/tilt)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/rtomayko/tilt/compare/v2.0.7...v2.0.8).

These are the first 10 commits:
- (a921750) [Update .travis.yml (#314)](https://github.com/rtomayko/tilt/commit/a92175095a97c399c40479b6689ab814ddf4d6f4)
- (51d9a3f) [Register .tsx for TypeScript (fixes #315)](https://github.com/rtomayko/tilt/commit/51d9a3f085552b3dfb27f5d614dff14c2d7e1dd6)
- (90c603c) [Use Haml 5's new API to avoid using deprecated private metho…](https://github.com/rtomayko/tilt/commit/90c603c6539c38e596efb85dff2e5d0e3bf68057)
- (6eea43e) [Travis: Install pandoc](https://github.com/rtomayko/tilt/commit/6eea43eab212d4533a28ee563f9ce2d64d887f02)
- (9f0d3c3) [Haml doesn't work on 1.9](https://github.com/rtomayko/tilt/commit/9f0d3c330c3e96ca7d96b5fb2807b32be18b32f2)
- (c037ff3) [Remove rubysl dependency](https://github.com/rtomayko/tilt/commit/c037ff307e4dac756cc35da78c75710f17891a06)
- (f928612) [Add gemspec in Gemfile:](https://github.com/rtomayko/tilt/commit/f928612e94c3875c91b6d8bd85e0d4a854ea9fb7)
- (727cd5a) [Don't bother with 1.9 and 2.0:](https://github.com/rtomayko/tilt/commit/727cd5a554e6f16d59e394657d48cc0d1bf01135)
- (c0c6e6e) [Drop JRuby 1.9 as well](https://github.com/rtomayko/tilt/commit/c0c6e6e1dc498433115d918e3fbf76ea139f953a)
- (924e5a0) [Uppercase option for CommonMarker render (#320)](https://github.com/rtomayko/tilt/commit/924e5a0b511a076457394b250db12c925a964424)


</details>